app/lib/widgets/authorization/view/auth.view.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:pipercrux/widgets/authorization/model/auth.model.dart';
```

Comment:
it's actually authentication, which means acknowledging the user
not authorization, which means verifying user's permission to do smth

i mean just rename folder